src/components/UI/Home_Header.jsx

```diff
@@ -1,9 +1,36 @@
+import { motion } from "framer-motion";
+import "./Home_Header.scss"; // ? Keep each component's SCSS truly separate
+
 const Header = () => {
   return (
-    <header id="home-header">
-      <button id="signUp-btn">sign up!</button>
-      <button id="login-btn">log in!</button>
-    </header>
+    <motion.header
```

Comment:
what is the difference between using motion.button on the element every where but here on the header and you place it after with a matching id?